query/src/storages/fuse/fuse_table.rs

```diff
@@ -152,7 +152,7 @@ impl Table for FuseTable {
         ctx: Arc<QueryContext>,
         stream: SendableDataBlockStream,
     ) -> Result<SendableDataBlockStream> {
-        let log_entry_stream = self.append_trunks(ctx, stream).await?;
+        let log_entry_stream = self.append_chunks(ctx, stream).await?;
```

Comment:
thanks :)

Reply:
> thanks :)

https://github.com/datafuselabs/databend/pull/5194

Reply:
Got it